src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1446,6 +1446,9 @@ so that we can apply CSS-filters to change the arrow color in themes */
 	align-items: center;
 	cursor: pointer;
 }
+.setting-check input {
+	flex-shrink: 0,
```

Comment:
Ah right, it was invalid CSS: it's supposed to be a colon and not a comma. Hence why it's important to add GUI test.